index.php

```diff
@@ -127,6 +127,50 @@
                 <a class="read-more" href="https://usn.ubuntu.com/releases/ubuntu-18.04-lts/">Security Notices</a>
             </div>
         </section>
+        <section id="workflow" class="grey">
+            <div class="grid">
+                <div class="two-thirds">
+                    <h2>Get work done. Or play.</h2>
+                    <h4>elementary OS is designed to keep you productive and focused by getting out of your way with Multitasking View, Picture-in-Picture, Do Not Disturb, and more. And by staying out of your way, elementary OS is great when you're watching videos or playing games, too.</h4>
+                </div>
+            </div>
+            <div class="grid">
+                <div class="third">
+                    <figure class="multitasking">
+                        <div class="workspace"></div>
+                    </figure>
+                    <h3>Multitasking View</h3>
+                    <p>Powerful workspaces help you organize your work by task. Keep work and play separate, but just one tap away.</p>
```

Comment:
Using the word "Powerful" is a pet peeve. What makes them "powerful"?

```suggestion
                    <p>Workspaces help organize your work by task. Keep work and play separate, but just one tap away.</p>
```